### DISCUSSION TITLE
help update Sonarr to v4

### DISCUSSION THREAD

**Discussion by Crash82** [swizzin/swizzin#1119] [Q&A + Help] [answered]
Is there a way to install Sonarr v4 or update to it from v3?

Sonarr keep saying im on version 3.0.10.1567 and v4 released Dec 2023 and I can't find any ways to update to v4.

So if there is anyone who knows how to, please let me know.

Thanks in advance.

**Comment by flying-sausages** [comment, marked as answer]
`box update` should do the trick
